pages/docs/tutorials/message-validation.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Message validation
+description: In this tutorial, you'll learn how to validate AsyncAPI messages (events).

```

Comment:
I think we can just stick to `message` and if you want in intro just reference message concept document where both are explained, message and event